packages/quick-open/src/browser/quick-open.view.tsx

```
@@ -79,25 +93,42 @@ export const QuickOpenHeader = observer(() => {
-        ))}
-      </div>
-    </div>
-  ) : null;
-});
+  const headerComponent = React.useMemo(() => {
+    if (leftButtons.length > 0 || titleText || rightButtons.length > 0) {
+      return (
+        <div className={styles.title_bar}>
+          <div className={styles.title_bar_button}>
+            {leftButtons.map((button) => (
+              <QuickOpenButton onMouseDown={(event) => onSelectButton(event, button)} button={button} />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)